apache/zeppelin
# [Zeppelin-1089] Replace JsHint with Eslint

### What is this PR for?
This PR replace JsHint with Eslint, the original configuration was ported from jsHint using [Polyjuice](https://www.npmjs.com/package/polyjuice), then the settings were tuned to match exactly.
For this PR, the goal is not to add new rules, but only to replace the system with the same rules.

### What type of PR is it?
Improvement

### Todos
* [x] - Remove .jshintrc

### What is the Jira issue?
https://issues.apache.org/jira/browse/ZEPPELIN-1089

### How should this be tested?
You can simply build the code.
You can try to add an unused variable in the js code to see an error.

### Questions:
* Does the licenses files need update? No
* Is there breaking changes for older versions? No
* Does this needs documentation? No
